src/services/OfficeRnDService.ts

```diff
@@ -97,25 +97,24 @@ export class OfficeRnDService {
     return keyBy(floorsArray.body, '_id');
   };
 
-  private getMembers = async () => {
+  private getTeams = async () => {
```

Comment:
It's better to make a fetch to retrieve specific teams
getTeamFromTeamIds = async(teamIds: string[]) => {
 ...